source/compute/portal/ondemand/jupyterlab.rst

```diff
@@ -37,19 +37,35 @@ drop-down menu.  An example would be '2023a and
 JupyterLab module will be loaded together with its dependencies (such as the
 listed Python module).
 
-Furthermore, you may choose to tick one of the checkboxes to load popular
-modules from the same toolchain, such as ``SciPy-bundle`` (for widely used
-packages like ``scipy``, ``numpy``, ``pandas`` and more) and/or ``matplotlib``.
+.. tab-set::
+   :sync-group: vsc-sites
+
+   .. tab-item:: KU Leuven/UHasselt
+      :sync: kuluh
+
+      Furthermore, you may choose to tick one of the checkboxes to load popular
+      modules from the same toolchain, such as ``SciPy-bundle`` (for widely used
+      packages like ``scipy``, ``numpy``, ``pandas`` and more) and/or ``matplotlib``.
+
+   .. tab-item:: VUB
+      :sync: vub
+
+      A large number of widely used Python packages are available via the
+      following modules, which are loaded by default:
+
+      - ``Python-bundle-PyPI`` - packages for general usage
+      - ``Scipy-bundle`` - data science packages like ``scipy``, ``numpy``, ``pandas``
+      - ``matplotlib`` - plotting library and :ref:`matplotlib_lab_extension`
+
 
-Once you launch a JupyterLab session, a default kernel called ``Python 3
-(ipykernel)`` is already available in your session.  This kernel, in addition to
-the Python standard library, would enable using extra packages from
-``SciPy-bundle`` and/or ``matplotlib``, if you selected them in the resource
-form.
+If you need additional Python packages, you can load extra modules with Python
+packages via ``module load`` commands in the 'Pre-run Scriptlet' of the
+resources form.
```

Comment:
Why not just use the `Lmod` panel in JupyterLab? that is much easier

Reply:
leuven doesn't have the modules extension, and if you regularly load the same modules it's easier in the pre-run scriptlet, because OOD remembers that list for you.

but i'll add it as alternative 👍